docs/dev-setup.md

```diff
@@ -71,6 +71,31 @@ Note that this script may need customizations to some of the `podman run`
 commands, to include environment variables that configure the containers for
 your environment.
 
+## Use with Redfish BMCs and UEFI
```

Comment:
This doesn't feel like "developer setup" documentation. I don't really see another obviously good place to put it. Maybe it's time to start a user guide? We can do that in a follow-up PR, unless someone points out an existing file that's a better home.

Reply:
I had the exact same thought, but proposed it as this to see what you guys thought.

Reply:
Not sure. There is a lot of background technical information here, which is useful to keep. I do agree on a user guide, but I think the background information can stay here, and the user guide should only mention the behavior we have to stick to.